models/SinglePendulum/SinglePendulum.jl

```diff
@@ -20,15 +17,17 @@
 # \ddot\theta = \dfrac{g}{L} \sin\theta + \dfrac{1}{m L^2} (T - c\dot\theta)
 # ```
 # where ``\theta`` is the angle that link makes with the upward vertical axis.
-# The state vector is ``[\theta, ̇\dot\theta]``.
+# The state vector is ``[θ, ̇θ̇]``.
 # Controllers are trained using behavior cloning. Here, a neural network is
-# trained to replicate expert demonstrations.
+# trained to replicate expert demonstrations [^1].
 
 using NeuralNetworkAnalysis
 
-controller = read_nnet("./controller_single_pendulum.nnet");
+models_dir = normpath(@__DIR__, "..", "..", "..", "models")
+path = joinpath(models_dir, "SinglePendulum", "controller_single_pendulum.nnet")
```

Comment:
What does this do? I think we should have a macro like `@relpath` in `ReachabilityModels`.

Reply:
yes, that would be convenient..